bmc-log-collector/bmc-http-client.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"crypto/tls"
+	//"errors"
+	"fmt"
+	"io"
+	"log/slog"
+	"net/http"
+	"os"
+	"time"
+)
+
+// タイムアウトが必要？
+// Get from Redfish on iDRAC webserver
+func bmcClient(url string) ([]byte, error) {
+	fmt.Println("client start to ", url)
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+
+	// タイムアウトの処理エラーのセットが欲しい
+	client := &http.Client{Timeout: time.Duration(10) * time.Second, Transport: tr}
```

Comment:
Transport has several timeouts. IdleConnTimeout, TLSHandshakeTimeout, ResponseHeaderTimeout etc.
Is this enough?